docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
```

Comment:
Yes, it could be a system user. I don't really see if/how this would make any practical difference, but I don't have any strong opinion about this. Do you have a proposal for a new UID/GUID?

I intentionally made this a build arg, because sometimes it's more comfortable to change the UID/GUID for running in production (file permissions, etc)